source/apps/deploy-github.txt

```diff
@@ -45,160 +51,269 @@ Prerequisites
+      .. procedure::
+      
+         .. step:: Authenticate a MongoDB Atlas User
 
-      Refer to the :ref:`app-configuration` reference page and individual component
-      reference pages for details on the structure and schema of your application
-      directory.
+            .. include:: /includes/cli-login.rst
 
-      .. important:: Do not make breaking schema changes via automated deploy
+         .. step:: Run the Deploy Configure Command
 
-         Because :ref:`breaking - also called destructive - schema changes
-         <destructive-changes-synced-schema>` require special handling, App Services
-         does not support making breaking schema changes via automated deploy
-         with GitHub. Instead, you should make breaking changes from the
-         App Services UI.
+            The {+cli+} can walk you through the process of configuring automatic
+            deployment. This is an interactive command.
 
+            .. code-block:: shell
 
-   .. step:: Commit and Push Your Changes
+               {+cli-bin+} deploy configure
 
-      Once you have updated your application configuration, you can deploy the
-      updates as a new version of your App by pushing them to the GitHub repo
-      that you specified. If Automatic GitHub Deployment is enabled,
-      App Services immediately deploys the latest commit for the configured
-      branch and directory.
+            The command also supports additional flags that you can optionally 
+            include. For information on these flags, refer to 
+            :ref:`appservices-deploy-configure`.
 
-      When you are ready to deploy, commit all of the files that you want to include
-      and then push them to GitHub:
+            {+cli+} immediately enables automatic deployment when you complete
+            this command. You don't need to run the ``enable`` command as
+            an additional step.
 
-      .. code-block:: sh
+            .. tip:: This functionality relies on a GitHub beta feature
 
-         git add -A
-         git commit -m "<commit message>"
-         git push origin <branch name>
+               The ability to configure automatic deployment using {+cli+}
+               relies on a GitHub feature that is currently in beta. If 
+               GitHub makes breaking changes to this feature, you may be unable
+               to configure automatic deployment using the {+cli+}. In this
+               case, use the App Services UI or Admin API to configure 
+               automatic deployment.
 
-      Once you successfully push your changes to GitHub, App Services
-      immediately deploys a new version of your application that matches the state
-      of the latest commit. Client applications will automatically use the newest
-      version, so make sure that you also update your client code to use the new
-      version if necessary.
+               This only applies to configuring automatic deployment. If you 
```

Comment:
would it be accurate to say that this only applies to the ``appservices deploy configure`` command, and the ``deploy enable`` and ``deploy disable`` commands don't rely on the feature?

Reply:
Updating the verbiage to clarify details!